dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/cookies/CookieUtils.java

```diff
@@ -393,7 +390,7 @@ private static String convertV0SetCookie(HttpCookie cookie, boolean isRFC1123Dat
 
         if(ProductInfo.getBetaEdition()) {
             value = cookie.getAttribute("partitioned");
-            if (null != value) {
+            if (null != value && !value.equalsIgnoreCase("false")) {
```

Comment:
Why not just check for value.equalsIgnoreCase("true")?

Reply:
I set it to an empty thing is some cases, but I guess it could be set to "true" there as well. 
https://github.com/OpenLiberty/open-liberty/pull/27843/files#diff-d0a69db5d3911d95a2ca377b482ad2c1d9463356ed2a2a4d5e834e54151df4edR2950


My reason is that since this is just a attribute and doesn't need a value, the value can be anything. ("true", "", "dummyValue", etc). However, false is explicit in that you don't want it set.

Reply:
Customers could use "" or "true" if they're on servlet 6.

Reply:
ok that's fair